pom.xml

```diff
@@ -14,7 +14,7 @@
   </licenses>
 
   <properties>
-    <raml-module-builder.version>23.3.0</raml-module-builder.version>
+    <raml-module-builder.version>23.12.0</raml-module-builder.version>
```

Comment:
there is a newer version of RMB is available, now that we are upgrading, we might as well move to the latest version

Reply:
Done